Update dev wiki (#3249)
fecgov/fecfile-online: README.md

Reviewer: This wiki page no longer exists.

## Set up

> You can find the set up instructions for the frontend of this project in our [wiki](https://github.com/fecgov/fecfile-web-api/wiki/Local-Setup).

Reviewer: What about something more like:
```suggestion
Set up instructions are found in the [wiki](https://github.com/fecgov/fecfile-web-api/wiki/Local-Setup), including [instructions specific to the frontend](https://github.com/fecgov/fecfile-web-api/wiki/Local-setup#running-the-frontend-locally).
```
